api/urls.py

```diff
@@ -1,16 +1,14 @@
 from flask_restful import Api
-from .resources import *
+from .resources import CategoryListResource, CategoryDetailResource, BotListOrCreateResource, BotResource, \
+    SignUpUserResource, UserLoginResource, UsersListResource
 from app import app
 
 api = Api(app)
 
-api.add_resource(CategoryListResource, '/categories')
+api.add_resource(CategoryListResource, '/api/category')
```

Comment:
I would not use such prefixes as api. it's kind of obvious anyways, but makes things longer